packages/suite/src/views/dashboard/components/PortfolioCard/components/EmptyWallet.tsx

```diff
@@ -35,15 +35,6 @@ const StyledImage = styled(Image)`
     margin-bottom: 24px;
 `;
 
-const SecurityItem = styled.div`
```

Comment:
cc @jvaclavik Should I keep this margins here or I can get rid of it?

Reply:
No 

![image](https://github.com/user-attachments/assets/410db664-e7aa-443f-94fe-46e78db688df)

Reply:
If `SecurityItem` component isn't used anywhere else in this file (looks like it isn't), you can remove this as you've already done. Looks like just a styled div, styled specifically for one specific sentence.

Reply:
If you are asking about 
```
   & + & {
        margin-top: 12px;
    }
```
this is a type of styling we are trying to get rid of from our codebase as it's less readable etc. This specifically styles siblings of an element. IMHO really no reason to keep it here. If we miss some margin somewhere close to this component, we should style it the other way.